src/TimeGutter.js

```diff
@@ -37,9 +39,9 @@ export default class TimeGutter extends Component {
 
   render() {
     const { resource, components, getters } = this.props
-
+    const { timeGutterWrapper: Wrapper = FallbackWrapper } = components
     return (
-      <div className="rbc-time-gutter rbc-time-column">
```

Comment:
I think we can keep this div as is and create a wrapper with default </> around it? If someone wants to override the class names, they will be able to.